modules/nf-core/ganon/buildcustom/tests/main.nf.test

```diff
@@ -9,17 +9,21 @@ nextflow_process {
     tag "modules_nfcore"
     tag "ganon"
     tag "ganon/buildcustom"
+    tag "temp"
```

Comment:
Is there a reason for tag temp?

Reply:
Debugging code I forgot to remove! Good catch !